sudokuProject/.classpath

```diff
@@ -4,5 +4,6 @@
 	<classpathentry kind="src" path="test"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
+	<classpathentry kind="lib" path="C:/Users/MiguelVega/workspace/Sudoku/sudokuProject/libs/javacsv.jar"/>
```

Comment:
if you are going to use a third party library make sure to add it inside your program and make a relative reference instead of a harcoded to your environment.